app/views/groups/members/_current_user_is_member.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= viral_dropdown(icon: "ellipsis_vertical",
+                                     classes: "member-settings-ellipsis",
```

Comment:
**question:** Why do you need this class?

Reply:
This was added pretty much at the beginning of when the members pages were developed. It was used to get the ellipsis in the tests. However we no longer use the user defined classes/ids to find stuff on the page. I've removed them in 09df5454052849947625f43e56f308b9b188e427